.github/workflows/push_docker_image.yml

```diff
@@ -0,0 +1,23 @@
+name: push docker image
+
+on: 
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: login
+        run: docker login --username '${{ secrets.DOCKER_USERNAME }}' --password '${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}'
+      - name: build
+        run: docker build . -t planetariumhq/ninechronicles-headless:git-${{ github.sha }} --build-arg COMMIT=git-${{ github.sha }} -t planetariumhq/ninechronicles-headless:latest --build-arg COMMIT=git-${{ github.sha }}
```

Comment:
I had thought since I started to review this PR. The build argument `COMMIT` is used to specify the version suffix of `NineChronicles.Standalone.Executable`. I wonder we need to specify the versioning policy of it.
When I read [https://semver.org](https://semver.org) again, it seems to present build metadata in [another way](https://semver.org/lang/ko/#spec-item-10) but ours haven't. So I thought it will be good that we should discuss it but you can feel that it is early or doesn't need to concern about it. I wonder your opinions @planetarium/libplanet.

Reply:
I guess it would be better if we changed version suffix from `1.0.0-<GIT-SHA>` to `1.0.0+<GIT-SHA>` to follow semver. (even if another PR)